var/spack/repos/builtin/packages/spectrum-mpi/package.py

```diff
@@ -0,0 +1,57 @@
+from spack import *
+
+class Spectrum(Package):
+    """IBM MPI implementation from Spectrum MPI."""
+
+    homepage = "http://www.example.com"
+    url      = "http://www.example.com/ibm-mpi-10.1.0.tar.gz"
+
+    version('10.1.0.2', '0123456789abcdef0123456789abcdef')
+    version('10.1.0', '0123456789abcdef0123456789abcdef')
```

Comment:
I ran spack flake8 and I fix all the complains.  Probably I did it after I added to git.  I thought git add will only add a reference and not the actual content of the file. Will redo.